tests/cross/src/test/scala/tests/pc/CompletionCaseSuite.scala

```diff
@@ -172,6 +170,12 @@ class CompletionCaseSuite extends BaseCompletionSuite {
     """|case scala.util.Left(value) =>
        |case Right(value) => scala.util
        |""".stripMargin,
+    compat = Map(
+      "3" ->
+        """|case Left(value) => scala.util
```

Comment:
```suggestion
        """|case scala.util.Left((value) => scala.util
```
Left has a conflicting name in scope, we can fix it later, but let's create an issue in that case

Reply:
Let's create a follow-up ticket and fix it in another PR :+1: